### ISSUE TITLE
Possible solution to beat and run angles

### ISSUE THREAD

**Issue by joabakk** [tkurki/polars-plugin#2]
In http://jieter.github.io/orc-data/site/ there is  possiblity to make an extended CSV, and this includes beat and run angles. The same logic could be applied here, so that one could copy and paste. Zeroes seem to mean "ignore me"

**Comment by tkurki** [comment]
Yeah I noticed https://github.com/jieter/orc-data/issues/7. Need to add support for it..
